daemon/config/config.go

```diff
@@ -169,7 +169,7 @@ func (cfg *Config) MergeConfigurations(flagSet *pflag.FlagSet) error {
 	}
 
 	fileFlags := make(map[string]interface{}, 0)
-	iterateConfig(origin, fileFlags)
+	iterateTLSConfig(origin, fileFlags)
 
 	// check if invalid or unknown flag exist in config file
 	if err = getUnknownFlags(flagSet, fileFlags); err != nil {
```

Comment:
in your commit, the `UnknownFlags` is only for TLS?

Reply:
`iterateTLSConfig` function is to make TLS value from map become a new key-value
```
{
   "TLS": {
       "a": "b"
    }
}  ->

{
   "a": "b"
}
```
as for other config, they will stay same, so do you got the point? @fuweid